README.md

```diff
@@ -74,6 +74,14 @@ add_filter( 'svg_allowed_tags', function ( $tags ) {
 } );
 ```
 
+### Can `wp_kses` be used with a helper to sanitize an SVG?
```

Comment:
**issue:** kses will not sanitise SVGs, so we either need to change this title, or re-think the use-case for this filter.